site/_data/templates.yaml

```diff
@@ -127,6 +130,36 @@
     - name: user
       type: User
       description: The User object corresponding to the authenticated user.
+# Account add WebAuthn passkey
+- displayName: Account add WebAuthn passkey
+  fieldName: accountWebAuthnAdd
+  description: This page contains a form that allows a user to register a new WebAuthn passkey.
+  path: /account/webauthn/add
+  version: 1.41.0
+  variables:
+    - name: user
+      type: User
+      description: The User object corresponding to the authenticated user.
+# Account delete WebAuthn passkey
+- displayName: Account delete WebAuthn passkey
+  fieldName: accountWebAuthnDelete
+  description: This page contains a form that allows a user to delete a WebAuthn passkey.
+  path: /account/webauthn/delete
+  version: 1.41.0
+  variables:
+    - name: credential
+      type: WebAuthnCredential
```

Comment:
Can we put a link in to the credentials object definition, like we did for the identity provider links (under `/oauth2/start-idp-link`)?

Reply:
Oops, I see you have a doc task to add that.

Reply:
Yeah. I have a task to add the link after I've created WebAuthn API doc page.